inst/extdata/run_seeker_array.sh

```diff
@@ -5,5 +5,5 @@ docker run \
   -w /home/rstudio/projects \
   --rm \
   ghcr.io/hugheylab/socker \
-  bash -c "Rscript run_seeker_array.R GSE25585.yml ." \
+  bash -c "Rscript -e "'"'"seeker::seekerArray('GSE25585', 'ensembl', parentDir = '.')"'"' \
```

Comment:
Nope, too opaque. I still want the shell script to run an R script that loads a yaml file.